versioned_docs/version-22.10/graph-views/map-web-update.md

```diff
@@ -68,9 +94,17 @@ sudo yum update centreon-map-engine centreon-map-web-client
 </Tabs>
 
 3. Clear your browser cache.
- 
-4. Restart the **centreon-map-engine** service using the following command:
+
+4. Finalize the update of the module and the widget in the Centreon interface **Administration > Extensions > Manager**.
+
+5. Restart the **centreon-map-engine** service using the following command:
  
   ```shell
   sudo systemctl start centreon-map-engine
   ```
+
+6. If you are using MAP Legacy, enter this command on the MAP server to apply any customization.
+
+ ```shell
+ diff -u /etc/centreon-studio/centreon-map.conf /etc/centreon-studio/centreon-map.conf.rpmsave
```

Comment:
This command only works for EL7/EL8/EL9 but not on Debian.
Debian will not create .rpmsave files
Also, if we update Centreon Map, we don't care about /etc/centreon-studio/ folder because it's related to centreon Map Legacy and the update will not modify these files